src/database/sql_database.cc

```diff
@@ -1388,9 +1388,14 @@ std::shared_ptr<CdsObject> SQLDatabase::createObjectFromSearchRow(const std::uni
     obj->setClass(getCol(row, SearchCol::upnp_class));
 
     auto meta = retrieveMetadataForObject(obj->getID());
-    if (!meta.empty())
+    if (!meta.empty()) {
         obj->setMetadata(meta);
 
+        // Update tile from metadata if present
+        if (meta.count(MetadataHandler::getMetaFieldName(M_TITLE)))
```

Comment:
find is a bool, no?

Reply:
to clarify, find() is faster than count() as it exits early.

Reply:
find() returns an iterator, so the proper form would be `meta.find() != meta.end()` IIRC. In the case of a std::map since all keys are unique I believe the performance would be equivalent.

Reply:
Sure no problem. I'll switch to find() later today